test/features/events.js

```diff
@@ -0,0 +1,312 @@
+'use strict';
+
+var Event2 = require('../../api/models/Event');
+
+describe('Check In Event', () => {
+
+  Scenario('Non-Existent Event should be Created', () => {
+
+    var date = new Date(923423234);
+    var dateString = date.toISOString();
+    var payload = {
+      person_id: 123,
+      cid_id: 999,
+      operation: 'check in',
+      timestamp: dateString,
+      gender: 'm',
+      nationality: 'gbr',
+      centre: 'bigone'
+    };
+
+    before(function () {
+      global.testConfig.initializeBarrelsFixtures = false;
+    });
+
+    after(function () {
+      global.testConfig.initializeBarrelsFixtures = true;
+    });
+
+    Given('a detainee event with timestamp "' + dateString + '" has not already occurred', () =>
```

Comment:
having a changing date in the test description will make it impossible to track failing tests either pick a fixed date or say 'now' in the given text